plugins/services/src/js/containers/services/ServicesTable.js

```diff
@@ -77,6 +61,75 @@ const METHODS_TO_BIND = [
   "handleSortClick"
 ];
 
+const serviceToVersion = serviceTreeNode => {
+  if (serviceTreeNode instanceof ServiceTree) {
+    return "";
+  }
+
+  return pipe(
+    Version.fromService,
+    Version.toDisplayVersion
+  )(serviceTreeNode);
+};
+
+function sortForColumn(name) {
```

Comment:
I think it would be better to call this `columnName` to avoid confusion?

Reply:
i'll gladly change that in a follow up